runtime/util/modularityHelper.c

```diff
@@ -36,11 +36,7 @@ static BOOLEAN isPackageExportedToModuleHelper(J9VMThread *currentThread, J9Modu
 BOOLEAN
 isModuleUnnamed(J9VMThread *currentThread, j9object_t moduleObject)
```

Comment:
I think this method can be folded as well.

Reply:
I leave this one alone as it's used in multiple compilation units.